src/lab7-tests.md

```diff
@@ -37,7 +37,7 @@ Here is how the lines are represented in chapter 7:
      DocumentLayout()
        BlockLayout(x=13, y=18, width=774, height=45.0)
          BlockLayout(x=13, y=18, width=774, height=45.0)
-           LineLayout(x=13, y=18, width=774, height=45.0)
+           InlineLayout(x=13, y=18, width=774, height=45.0)
```

Comment:
Is this because of the lab6 change to BlockLayout?

Reply:
Nah, the `__repr__` for `InlineLayout` called itself `LineLayout`; I assume it's a copy-and-paste error.